jobs/views.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework import viewsets, status, mixins
+from rest_framework.response import Response
+from .models import Job, Position
+from .serializers import JobCardSerializer, JobDetailSerializer, PositionSerializer
+from .utils import check_user_owner
+
+
+class PositionViewSet(mixins.RetrieveModelMixin,
+                      mixins.ListModelMixin,
+                      mixins.CreateModelMixin,
+                      viewsets.GenericViewSet):
+    queryset = Position.objects.all()
+    serializer_class = PositionSerializer
+
+
+class JobsViewSet(viewsets.ViewSet):
+    def get_jobs_card(self, request):
+        jobs = Job.objects.all()
+        serializer = JobCardSerializer(jobs, many=True)
+        return Response(serializer.data)
+
+    def post(self, request):
+        if check_user_owner(request):
```

Comment:
We can put this validation inside the `JobDetailSerializer`. We could write a `validate` method to validate the data of the request.